discover/portal/articles/140-workflow/00-intro.markdown

```diff
@@ -0,0 +1,136 @@
+# Workflow [](id=workflow)
+
+The ancient Greeks were pretty important, so we decided to name Liferay's
+workflow engine *Kaleo*. In Greek, Kaleo means "called ones," which is
+appropriate for a workflow engine that calls users to participate in a review
+process designed for them. Kaleo workflow allows a user to define any number of
+simple to complex business processes/workflows, deploy them, and manage them
+through a portal interface. The processes have knowledge of users, groups and
+roles. You don't have to write a single line of code to accomplish this: all
+you have to do is create one XML document, or if you're a Liferay Digital
+Experience Platform (DXP) customer, you have access to a nifty visual designer
+for workflows, called the Workflow Designer. 
+
+There are several aspects to workflow that you need to understand: 
+
+- Enabling workflow
+
+- Designing review processes
+
+- Uploading workflow definitions
+
+- Configuring assets to use workflows
+
+- Using workflow with other applications
+
+After all that, you should be familiar with how to use Liferay's Kaleo workflow
+to set up approval process for any kind of content before it is published to
```

Comment:
The content closer from what you intent to show that I could find is dev.liferay.com is https://dev.liferay.com/discover/portal/-/knowledge_base/7-0/enabling-workflow#workflow-assets.

Reply:
It's in your queue ;)
I've changed the name from Enabling Workflow to Activating Workflow, since I think it more accurately describes what you're doing. I think the development task of integrating the Workflow Framework with the asset is workflow-enabling, and actually specifying that an asset uses a specific workflow is Activating Workflow on the asset. So it's already enabled for workflow, but to use it you must activate it. Make sense?

Reply:
For me makes total sense! In the UI this is handled as Workflow Assignment, but, I believe that Activating Workflow transmits the idea. 😉